[2024-05-14T03:32:27.051+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T03:32:27.069+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: MI_PRIMER_DAG.third_task manual__2024-05-14T03:32:22.420838+00:00 [queued]>
[2024-05-14T03:32:27.076+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: MI_PRIMER_DAG.third_task manual__2024-05-14T03:32:22.420838+00:00 [queued]>
[2024-05-14T03:32:27.077+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 5
[2024-05-14T03:32:27.090+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): third_task> on 2024-05-14 03:32:22.420838+00:00
[2024-05-14T03:32:27.095+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1751) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T03:32:27.096+0000] {standard_task_runner.py:63} INFO - Started process 1753 to run task
[2024-05-14T03:32:27.096+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'MI_PRIMER_DAG', 'third_task', 'manual__2024-05-14T03:32:22.420838+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/mi_primer_dag.py', '--cfg-path', '/tmp/tmpfuqrgqz7']
[2024-05-14T03:32:27.098+0000] {standard_task_runner.py:91} INFO - Job 40: Subtask third_task
[2024-05-14T03:32:27.140+0000] {task_command.py:426} INFO - Running <TaskInstance: MI_PRIMER_DAG.third_task manual__2024-05-14T03:32:22.420838+00:00 [running]> on host 33978f229c7f
[2024-05-14T03:32:27.276+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='MI_PRIMER_DAG' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T03:32:22.420838+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T03:32:22.420838+00:00'
[2024-05-14T03:32:27.278+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T03:32:27.278+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-14T03:32:27.279+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T03:32:27.280+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T03:32:27.290+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=MI_PRIMER_DAG, task_id=third_task, execution_date=20240514T033222, start_date=20240514T033227, end_date=20240514T033227
[2024-05-14T03:32:27.310+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T03:32:27.331+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-14T03:32:27.334+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
